### PR TITLE
Fix licenses typos

### PR DESCRIPTION
**Pull request by MrSidims**
Signed-off-by: Dmitry Sidorov <dmitry.sidorov@intel.com>

### CODE REVIEW

**Comment by MrSidims** [comment]
@svenvh @StuartDBrady @AlexeySotkin @AlexeySachkov please take a look.
Would be better to use the one from https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/80 tho.

**Comment by svenvh** [comment]
> @svenvh @StuartDBrady @AlexeySotkin @AlexeySachkov please take a look.
> Would be better to use the one from #80 tho.

I fully agree that it would be better to move to the new license / copyright headers indeed, but this has been a long-standing issue and there hasn't been much progress unfortunately. Until we have a clear path forward, I'm very reluctant to approve any changes to the current headers.

File: include/LLVMSPIRVOpts.h [hunk 1]
```diff
@@ -19,9 +19,9 @@
 // Redistributions in binary form must reproduce the above copyright notice,
 // this list of conditions and the following disclaimers in the documentation
 // and/or other materials provided with the distribution.
-// Neither the names of Advanced Micro Devices, Inc., nor the names of its
-// contributors may be used to endorse or promote products derived from this
-// Software without specific prior written permission.
+// Neither the names of Intel Corporation, nor the names of its contributors may
```
Comment (mlychkov) [inline include/LLVMSPIRVOpts.h:22]: What about LLVMSPIRVOpts.cpp?

Comment (MrSidims) [reply include/LLVMSPIRVOpts.h:22]: Missed it during grepping, thanks!

**Comment by MrSidims** [comment]
> > @svenvh @StuartDBrady @AlexeySotkin @AlexeySachkov please take a look.
> > Would be better to use the one from #80 tho.
> 
> I fully agree that it would be better to move to the new license / copyright headers indeed, but this has been a long-standing issue and there hasn't been much progress unfortunately. Until we have a clear path forward, I'm very reluctant to approve any changes to the current headers.

Agree. This PR addresses a different issue, it's a bit weird to see in the same file  `// Copyright (c) 2019 Intel Corporation. All rights reserved.` (as the file was added by @AlexeySachkov) and few lines later `AMD`, which does look like a copy-paste error.

**Pull request closed without merging**